front/public/locales/en/common.json

```
@@ -10,5 +10,11 @@
       "en": "English",
       "fr": "French"
     }
+  },
+  "productPage": {
+    "similarProducts": "Similar products",
+    "onStock": "On stock",
```

Don't forget to change this in other occurences

```suggestion
    "inStock": "In stock",
```

---

will be pushed soon